dev.fredol.open-tv.yaml

```diff
@@ -0,0 +1,628 @@
+            url: https://github.com/google/shaderc.git
+            #tag: v2023.7
+            commit: 40bced4e1e205ecf44630d2dfa357655b6dabd04
+            #x-checker-data:
+            #  type: git
+            #  tag-pattern: ^v(\d{4}\.\d{1,2})$
+          - type: git
+            url: https://github.com/KhronosGroup/SPIRV-Tools.git
+            tag: v2024.1
+            commit: 04896c462d9f3f504c99a4698605b6524af813c1
+            dest: third_party/spirv-tools
+            #x-checker-data:
+            #  type: git
+            #  tag-pattern: ^v(\d{4}\.\d{1})$
+          - type: git
+            url: https://github.com/KhronosGroup/SPIRV-Headers.git
+            #tag: sdk-1.3.250.1
+            commit: 4f7b471f1a66b6d06462cd4ba57628cc0cd087d7
+            dest: third_party/spirv-headers
+            #x-checker-data:
+            #  type: git
+            #  tag-pattern: ^sdk-([\d.]+)$
+          - type: git
+            url: https://github.com/KhronosGroup/glslang.git
+            tag: 15.0.0
+            commit: 46ef757e048e760b46601e6e77ae0cb72c97bd2f
+            dest: third_party/glslang
+            x-checker-data:
+              type: git
+              tag-pattern: ^(\d{1,2}\.\d{1,2}\.\d{1,4})$
+
+  - name: mpv
+    buildsystem: meson
+    config-opts:
+      - -Dbuild-date=false
+      - -Dlibmpv=true
+      - -Dmanpage-build=disabled
+      - -Ddvbin=enabled
+      - -Dlibarchive=enabled
+      - -Dsdl2=enabled
+      #- -Dshaderc=enabled
+      - -Dvulkan=enabled
+      # - -Dvulkan-interop=enabled
+    cleanup:
+      - /include
+      - /lib/pkgconfig
+    post-install:
+      - echo "screenshot-directory=~/.var/app/dev.fredol.open-tv/Pictures/mpv" > /app/etc/mpv/mpv.conf
+    sources:
+      - type: git
+        url: https://github.com/mpv-player/mpv.git
+        tag: v0.39.0
+        commit: a0fba7be57f3822d967b04f0f6b6d6341e7516e7
+        x-checker-data:
+          type: git
+          tag-pattern: ^v([\d.]+)$
+
+  - name: mpv-mpris
+    no-autogen: true
+    make-install-args:
+      - SCRIPTS_DIR=/app/etc/mpv/scripts
+    sources:
+      - type: archive
+        archive-type: tar
+        url: https://api.github.com/repos/hoyon/mpv-mpris/tarball/1.1
+        sha256: 08d6b53a41224710ebed1c4d6daee815686e0f2f10e3f81778f4411562ed5958
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/hoyon/mpv-mpris/releases/latest
+          version-query: .name
+          url-query: .tarball_url
+
+  # This mpv plugin prevents screen blanking in GNOME (wayland) while playing media.
+  - name: mpv_inhibit_gnome
+    buildsystem: simple
+    build-commands:
+      - make
+      - install -D lib/mpv_inhibit_gnome.so /app/etc/mpv/scripts
+    sources:
+      - type: git
+        url: https://github.com/Guldoman/mpv_inhibit_gnome.git
+        tag: v0.1.3
+        commit: 9069be0f15fa2c222d78a183d4aa1477d8a8f2cd
+        x-checker-data:
+          type: git
+          tag-pattern: ^v([\d.]+)$
```

Comment:
Are these two actually needed for your app? These are MPV plugins.

Reply:
my app pretty much invokes mpv, so this plugin could be useful to gnome users